native/helloworld-rs/README.md

```diff
@@ -1,27 +1,35 @@
-# Simple "Hello, world!" Rust Program on Unikraft
+# Rust "Hello, World!"
 
-This directory contains an example Rust-based "Hello, world!" example.
+This directory contains a Rust "Hello, World!" example running on Unikraft.
 
-To run this example, install [Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli) and [the `nightly` Rust toolchain channel through Rustup](https://www.rust-lang.org/tools/install).
+To run this example, install [Unikraft's companion command-line toolchain `kraft`](https://unikraft.org/docs/cli) and [the `nightly` Rust toolchain channel through `Rustup`](https://www.rust-lang.org/tools/install).
```

Comment:
Rustup should either be without backticks and capitalized when used as a proper name, or not capitalized and with backticks if used as a command.